pyvrp/search/LocalSearch.py

```diff
@@ -76,7 +76,7 @@ def get_neighbours(self) -> List[List[int]]:
         """
         return self._ls.get_neighbours()
 
-    def run(
```

Comment:
Rather than changing `run` to `__call__`, can't we just pass `LocalSearch.run` to genetic algorithm which (I think) implements the `SearchMethod` protocol. The user would then (in theory) also have the option to pass `LocalSearch.search` or `LocalSearch.intensify`, as well as any other function (see also https://github.com/PyVRP/PyVRP/issues/46#issuecomment-1649599134).

Reply:
The user already has that option, because `search` and `intensify` also implement `SearchMethod`. So any of

- `GA(..., search_method=ls, ...)`
- `GA(..., search_method=ls.search, ...)`
- `GA(..., search_method=ls.intensiyf, ...)`

Should work. For `LocalSearch` to be a search method itself we do need `__call__` here, not `run`.